install_night.html

```diff
@@ -36,15 +36,43 @@
       <!-- Any section element inside of this container is displayed as a slide -->
       <div class="slides">
         <!-- Opening slide -->
+        <section data-background-iframe="https://developer.apple.com/xcode/downloads/">
+          <h2>Installing Xcode</h2>
+          <p>Head on over to: https://developer.apple.com/xcode/downloads/</p>
+          <img style="max-width:370px; min-width:330px;" src="https://devimages.apple.com.edgekey.net/xcode/images/downloads_2x.png">
+        </section>
         <section>
-          <h3>Welcome!</h3>
-          <p>Jeff, this is you buddy :) Setup Instructions for Install Night go here.</p>
+          <h2>CocoaPods</h2>
+          <ul>
+              <li>Manages library dependencies for Xcode projects</li>
+              <li>Dependencies are specified in a single text file called a Podfile</li>
+              <li>Resolves dependencies between libraries, fetches the required source code and links everything together in an Xcode workspace to build your project</li>
+          </ul>
+          <p>Reference: </p> <a href="https://guides.cocoapods.org/using/getting-started.html">https://guides.cocoapods.org/using/getting-started.html</a>
+
         </section>
         <section>
-          <p>Second Slide</p>
+          <h2>Installing CocoaPods</h2>
+          <ol>
+              <li>Open Terminal and enter:</li>
+              <blockquote style="max-width:700px; min-width:700px;">sudo gem install cocoapods</blockquote>
+          </ol>
+          <p>NOTE: CocoaPods is built with Ruby and it will be installable with the default Ruby available on OS X</p>
         </section>
         <section>
-          <p>Third Slide</p>
+          <h2>Using CocoaPods</h2>
+          <ol>
+              <li>Navigate to your project's folder, create a file called Podfile and open it in any editor</li>
```

Comment:
Do you think the students would be comfortable with terminal/VIM?  This
would be the only time they would need some sort of editor outside of
Xcode, so I'm not sure if it would be worthwhile to spend time on text
editor options.

On Tuesday, May 12, 2015, Stephanie <notifications@github.com> wrote:

> In install_night.html
> <https://github.com/gdiboston/gdi-intro-ios/pull/1#discussion_r30191163>:
>
> >          </section>
> >          <section>
> > -          <p>Third Slide</p>
> > +          <h2>Using CocoaPods</h2>
> > +          <ol>
> > +              <li>Navigate to your project's folder, create a file called Podfile and open it in any editor</li>
>
> Some students may need help with this part of the setup and we can either
> stick to using Finder or use the Terminal, whichever you prefer.
>
> This also makes me realize that these students may not have a text editor
> other than the Mac default TextEdit. Shall we show them a text editor they
> can download if we don't want them using TextEdit? I feel like most people
> (including TAs) will know Sublime basics, so perhaps that might be a good
> option to present.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdiboston/gdi-intro-ios/pull/1/files#r30191163>.
>


-- 
Best Regards,
Jeff O'Leary


[image: MeYou Health][image: FB] <https://www.facebook.com/meyouhealth>[image:
T] <http://twitter.com/#!/meyouhealth>[image: RSS]
<http://www.meyouhealth.com/blog/>[image: YT]
<http://www.youtube.com/user/meyouhealthvideos>[image: F]
<http://www.flickr.com/photos/meyouhealth>
*Jeff O'Leary*iOS Engineer
Cell: 857-268-2939
MeYou Health - the social well-being companywww.meyouhealth.com
jeff.oleary@meyouhealth.com
MeYou Health is a Healthways, Inc. company